src/frontend/next/src/components/SearchHelp/SEO.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import Head from 'next/head';
+import useSiteMetadata from '../hooks/use-site-metadata';
```

Comment:
The site-metadata hook isn't going to work as is, since it uses graphql to talk to Gatsby.  We'll have to figure out a plan for passing data into next components from the build statically, maybe as env variables or maybe using static props.  I know that others are thinking about this too.